python/dgl/distributed/sparse_emb.py

```diff
@@ -5,25 +5,52 @@
 from .dist_tensor import DistTensor
 
 class DistEmbedding:
-    '''Embeddings in the distributed training.
+    '''Distributed embeddings.
+
+    DGL provides a distributed embedding to support models that require learnable embeddings.
+    DGL's distributed embeddings are mainly used for learning node embeddings of graph models.
+    Because distributed embeddings are part of a model, they are updated by mini-batches.
+    The distributed embeddings have to be updated by DGL's optimizers instead of
+    the optimizers provided by the deep learning frameworks (e.g., Pytorch and MXNet).
+
+    To support efficient training on a graph with many nodes, the embeddings support sparse
+    updates. That is, only the embeddings involved in a mini-batch computation are updated.
+    Currently, DGL provides only one optimizer: `SparseAdagrad`. DGL will provide more
+    optimizers in the future.
+
+    Like distributed tensors, distributed embeddings are sharded and stored in a cluster
```

Comment:
The distributed embeddings are sharded and stored in a cluster of machines and shared almost the same behavior as DistTensor, except it is trainable.

We do not need the following words.